Error message when npm installing

```
module.js:340
    throw err;
          ^
Error: Cannot find module '../lib/index'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/doxx/bin/doxx:6:12)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
```

This is due to the filename in /lib.

--- a/bin/doxx
+++ b/bin/doxx
@@ -3,7 +3,7 @@
 
 var program = require('commander'),
     version = require('../package').version,
-    doxx = require('../lib/index'),
+    doxx = require('../lib/doxx'),
     defaults = require('../lib/defaults');
 
 /**

